packages/eslint-plugin/src/stylex-valid-styles.js

```diff
@@ -592,20 +592,9 @@ const borderImage = makeUnionRule(
   isString,
   borderImageRepeat,
 );
-// const borderInlineEnd = makeUnionRule(borderWidth, borderStyle, color);
-// const borderInlineEndColor = color;
-// const borderInlineEndStyle = borderStyle;
-// const borderInlineEndWidth = borderWidth;
-// const borderInlineStart = makeUnionRule(borderWidth, borderStyle, color);
-// const borderInlineStartColor = color;
-// const borderInlineStartStyle = borderStyle;
-// const borderInlineStartWidth = borderWidth;
 const borderLeftColor = color;
 const borderLeftStyle = brStyle;
 const borderLeftWidth = borderWidth;
-const borderRightColor = color;
```

Comment:
Are these meant to be deleted?

Reply:
Yeah, `borderRightColor` references `color` directly instead.